lpd/client.py

```diff
@@ -40,7 +40,7 @@ def _decompress_username(cls, username):
     @classmethod
     def send_learner_data(cls, user, scores):
         """
-        Sends POST request with up-to-date learner data for `user` to adaptive engine.
+        Sends PUT request with up-to-date learner data for `user` to adaptive engine.
```

Comment:
Good job spotting that typo :)

Reply:
@tomaszgy Yeah... I really should have updated this in https://github.com/open-craft/learner-profile-dashboard/pull/10, when adapting the logic for sending learner profile data according to the changes that Andrew made to the adaptive engine's endpoint ;)